client/main/variableeditor.js

```diff
@@ -221,15 +221,16 @@ const EditorWidget = Backbone.View.extend({
         this.$el.empty();
         this.$el.addClass('silky-variable-editor-widget');
 
-        this.$title = $('<input class="silky-variable-editor-widget-title" type="text">').appendTo(this.$el);
+        this.$title = $('<input class="silky-variable-editor-widget-title" type="text" maxlength="50">').appendTo(this.$el);
```

Comment:
hmm, but then you'll have to do a similar thing at the js level

might be easiest to have a 63 character limit at the js level. sure, that could end up being represented by 256 bytes, but who cares?

that way you can just use the `name = name[:63]`